frappe/public/js/frappe/db.js

```diff
@@ -31,14 +31,15 @@ frappe.db = {
 			});
 		});
 	},
-	get_value: function(doctype, filters, fieldname, callback) {
+	get_value: function(doctype, filters, fieldname, parent_doc, callback) {
```

Comment:
always add a new parameter in the end, as it will break for those who have used the old argument sequence

Reply:
Fixed